src/igraph/io/objects.py

```diff
@@ -697,31 +697,31 @@ def _export_graph_to_dict_dict(
 
     This function is the reverse of Graph.DictDict.
 
-    @param use_vids (bool): whether to label vertices in the output data
+    Example:
+
+      >>> g = Graph.Full(3)
+      >>> g.es['name'] = ['first_edge', 'second', 'third']
+      >>> g.to_dict_dict()
+      {0: {1: {'name': 'first_edge'}, 2: {'name': 'second'}}, 1: {2: {'name': 'third'}}}
+
+    @param use_vids: (bool) whether to label vertices in the output data
       structure by their ids or their vertex_name_attr attribute. If
       use_vids=False but vertices lack a vertex_name_attr attribute, an
       AttributeError is raised.
-    @param edge_attrs (str or list of str): list of edge attributes to export.
+    @param edge_attrs: (str or list of str) list of edge attributes to export.
```

Comment:
Again, type annotations should be removed from here or moved to the function header.

Reply:
yep